### PR TITLE
Badge button

### PR DESCRIPTION
**Pull request by archit7448**
completed button and Badges

### CODE REVIEW

File: Components/Buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.button{
+    padding: 0.5rem 1rem;
+    cursor: pointer;
+    border-radius: 2px;
+    background-color: white;
```
Comment (amartya-rs) [inline Components/Buttons/buttons.css]: You can use a css color variable here.

File: Main/Main.css [hunk 2]
```diff
@@ -6,8 +6,11 @@
     border: 0px;
 }
 :root {
-    --colour-primary : #2ecc71;
-    --colour-secondary: white;
+    --color-primary : #2ecc71;
```
Comment (dcs-soni) [inline Main/Main.css]: Specify which color is the primary color. This will confuse in the future

File: Main/Main.css [hunk 3]
```diff
@@ -6,8 +6,11 @@
...
 }
 :root {
-    --colour-primary : #2ecc71;
-    --colour-secondary: white;
+    --color-primary : #2ecc71;
+    --color-secondary: white;
+    --color-primary-bright:rgb(57,255,20);
+    --color-black:#000000bf;
+    --color-grey :grey;
     --alert-success:#60C060;
```
Comment (dcs-soni) [inline Main/Main.css:14]: Here also, specify which color is secondary, primary bright, etc.

**Review by dcs-soni — CHANGES_REQUESTED** [review]

File: Main/Main.css (hunk 3 above)
Comment (archit7448) [reply Main/Main.css:14]: I have update this thank you for the suggestion

File: Components/Buttons/buttons.css (hunk 1 above)
Comment (archit7448) [reply Components/Buttons/buttons.css]: I have update this

**Review by dcs-soni — APPROVED** [review]

**Review by tonymike99 — APPROVED** [review]
Class names are great. Good work!

**Pull request merged**